packages/connect-web/src/index.ts

```diff
@@ -51,3 +51,5 @@ export {
 } from "./http-headers.js";
 
 export { createEnvelopeReadableStream } from "./envelope.js";
+
+export { ReadableStreamDefaultReadResultLike } from "./lib.dom.streams.js";
```

Comment:
We must not export this. The user brings their own type. For example:

```ts
declare var a: StreamResponse<SayRequest>;
const b: ReadableStreamReadResult<SayRequest> = await a.read();
```

`r.read()` returns our `ReadableStreamDefaultReadResultLike`, but it's assignable to `ReadableStreamReadResult` and to `ReadableStreamDefaultReadResult`, which means our typings are compatible with both. As long as the types are assignable, we're good.